app/views/shared/_help_modal.html.erb

```diff
@@ -2,7 +2,7 @@
   <div class="modal-dialog modal-dialog-centered ">
     <div class="modal-content">
       <div class="modal-header">
-        <h3 class="modal-title" id="help-prompt">Help</h3>
+        <h1 class="modal-title" id="help-prompt">Help</h1>
```

Comment:
I wonder if it's OK for the size of this text to bump up significantly, or if we want to use the `<h1>` element for semantics and use CSS to size the text closer to that of an `<h3>`. I believe adding the `h3` CSS class might do this (via bootstrap)?

Reply:
Good suggestion @mjgiarlo fixed.